packages/common/src/utils/badges/PatronBadge.tsx

```diff
@@ -7,6 +7,10 @@ const DEFAULT_BADGE = {
   visible: true,
 };
 
-export default ({ size, ...props }: { size: number }) => (
-  <Badge {...props} badge={DEFAULT_BADGE} size={size} />
-);
+export default ({
+  size,
+  ...props
+}: {
```

Comment:
@SaraVieira I think it's better to have a separate `Props` type

```js
type Props = {
  size: number;
  style?: React.CSSProperties;
}

export default ({ size, ...props }: Props) =>
```

Reply:
Done and also removed the exported default for ts to recognize the name :)